composer.json

```diff
@@ -19,7 +19,7 @@
         "wp-cli/core-command": "^1 || ^2",
         "wp-cli/eval-command": "^1 || ^2",
         "wp-cli/wp-cli": "^2.5.1",
-        "wp-coding-standards/wpcs": "^2.3.0",
+        "wp-coding-standards/wpcs": "^3",
```

Comment:
Recommended: remove the explicit `dealerdirect/phpcodesniffer-composer-installer` require. That now comes with WPCS and removing it will prevent potential future version conflicts.